engine/access/rpc/engine.go

```diff
@@ -152,13 +156,17 @@ func New(log zerolog.Logger,
 		log,
 	)
 
+	restAPIHandler := rest.NewRestAPIHandler(backend, log)
+	restServer := rest.NewRestAPIServer(restAPIHandler, config.RESTListenAddr, log)
```

Comment:
Is the REST api enabled by default? Should be make this configurable at command line?

Reply:
agreed..added a flag